### ISSUE TITLE
Show a failed build check if PR title doesn't match

### ISSUE THREAD

**Issue by exocom** [MorrisonCole/pr-lint-action#111]
Hey is it possible to add a toggle to fail the build if the PR title doesn't match instead of adding a comment? IE make this configurable.

Our workflow is to look for failed builds and fix them. Currently this action always succeeds even if the branch does not match which is counter intuitive for us.

Ideally I would want to toggle them separately:

add-review: false
fail-on-error: true

**Comment by MorrisonCole** [comment]
Hey Kalarrs!

Yeah, I think that should be possible and it certainly seems to make sense - need to check the details and get back to you. It's on my TODO list for today!

**Comment by MorrisonCole** [comment]
@exocom I've just released [v1.3.0](

The main `README` is updated with all the details, but you can now configure your workflow (for example) like:

```yaml
steps:
  - uses: morrisoncole/pr-lint-action@v1.3.0
    with:
      title-regex: "#[eE][xX]-[0-9]+"
      on-failed-regex-fail-action: true
      on-failed-regex-create-review: false
      on-failed-regex-comment: "Error log for failed regex: `%regex%`!"
      repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

**Comment by MorrisonCole** [comment]
Gonna close this as fixed but feel free to let me know if you have any further requests!
